package-lock.json

```diff
@@ -45,7 +45,7 @@
                 "fast-xml-parser": "^4.0.11",
                 "fs-extra": "^9.1.0",
                 "glob": "^8.1.0",
-                "hereby": "^1.6.4",
+                "hereby": "^1.8.1",
```

Comment:
you probably shouldn't update hereby in this PR. Our overnight run regenerates package-lock.json to use the latest version.